### PR TITLE
AST-435 - imprv : remove HTML widget related static CSS and move it to dynamic with conditions

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Remove HTML widget related static CSS and move it to dynamic with conditions
- https://wp-astra.atlassian.net/browse/AST-435

### Types of changes
- Improvements

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/footer/html/dynamic-css/dynamic.css.php [hunk 1]
```diff
@@ -46,6 +46,9 @@ function astra_fb_html_dynamic_css( $dynamic_css, $dynamic_css_filtered = '' ) {
 		 * Copyright CSS.
 		 */
 		$css_output_desktop = array(
+			'.footer-widget-area[data-section^="section-fb-html-"] .ast-builder-html-element' => array(
+				'text-align' => 'center',
+			),
```
Comment (premanshup) [inline inc/builder/type/footer/html/dynamic-css/dynamic.css.php:51]: @sagarb90 Adding CSS here will add it in loop and multiple CSS will be added. Can you please check if we should keep this outside of loop?

File: sass/site/builder/_header.scss [hunk 2]
```diff
@@ -186,7 +165,7 @@
  * AST HF Builder - Widgets related CSS.
  * Social Widget.
  */
-.ast-builder-social-element {
+ .ast-builder-social-element {
```
Comment (premanshup) [inline sass/site/builder/_header.scss:168]:
```suggestion
.ast-builder-social-element {
```

File: sass/site/builder/_header.scss [hunk 3]
```diff
@@ -211,6 +190,7 @@
 		margin-bottom: 6px;
 	}
 }
+
```
Comment (premanshup) [inline sass/site/builder/_header.scss:193]:
```suggestion

```

File: sass/site/builder/_header.scss [hunk 4]
```diff
@@ -224,6 +204,7 @@
 		}
 	}
 }
+
```
Comment (premanshup) [inline sass/site/builder/_header.scss:207]:
```suggestion

```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/builder/type/footer/html/dynamic-css/dynamic.css.php (hunk 1 above)
Comment (sagarb90) [reply inc/builder/type/footer/html/dynamic-css/dynamic.css.php]: Yes added to outside of loop

Comment (premanshup) [reply inc/builder/type/footer/html/dynamic-css/dynamic.css.php]: Thank you!

**Review by premanshup — APPROVED** [review]

**Pull request merged**